src/components/views/right_panel/PinnedMessagesCard.tsx

```diff
@@ -196,7 +196,9 @@ const PinnedMessagesCard: React.FC<IProps> = ({ room, onClose, permalinkCreator
                         <div className="mx_MessageActionBar_iconButton">
                             <ReplyIcon />
                         </div>
-                        <div className="mx_MessageActionBar_iconButton mx_MessageActionBar_optionsButton">
+                        <div className="mx_MessageActionBar_iconButton mx_MessageActionBar_optionsButton"
+                            data-testid="pinned-messages-card-button-options"
```

Comment:
This intends to ensure that `.mx_MessageActionBar_iconButton.mx_MessageActionBar_optionsButton` on pinned messages card is not selected when it is opened on the right panel along with messages on the main timeline.